packages/core/src/html/linkProcessor.js

```diff
@@ -152,37 +143,25 @@ function isValidFileAsset(resourcePath, config) {
  * If the intra-links are suspected to be invalid and they do not have the no-validation
  * attribute, a warning message will be logged.
  *
- * @param {Object<any, any>} node from the dom traversal
+ * @param {String} resourcePath parsed from the node's relevant attribute
  * @param {string} cwf as flagged from {@link NodePreprocessor}
  * @param {Object<any, any>} config passed for page metadata access
  * @returns {string} these string return values are for unit testing purposes only
  */
-function validateIntraLink(node, cwf, config) {
-  if (!tagsToValidate.has(node.name)) {
-    return 'Should not validate';
-  }
-
-  if (node.attribs) {
-    const hasIntralinkValidationDisabled = lodashHas(node.attribs, 'no-validation');
-    if (hasIntralinkValidationDisabled) {
-      return 'Intralink validation disabled';
-    }
-  }
-
-  let resourcePath = getDefaultTagsResourcePath(node);
+function validateIntraLink(resourcePath, cwf, config) {
```

Comment:
We need to make sure for whatever calls this function will use the new definition too.